java/amazon-kinesis-producer-sample/pom.xml

```diff
@@ -46,11 +46,17 @@
             <groupId>com.amazonaws</groupId>
             <artifactId>amazon-kinesis-client</artifactId>
             <version>1.11.2</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.google.protobuf</groupId>
+                    <artifactId>protobuf-java</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>amazon-kinesis-producer</artifactId>
-            <version>0.14.0</version>
+            <version>0.14.0-SNAPSHOT</version>
```

Comment:
Since we plan on rolling out a new version with this, let's update this to 0.14.1